.editorconfig

```diff
@@ -0,0 +1,30 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
```

Comment:
I know gofmt will fix this, but go uses tabs not spaces. Might be good to set that in here. I am not sure if go cares what indent size is used for tabs. I have mostly seen it at 4 and gofmt might just do that anyway.

Consider adding a section for go, or adding to an existing one where makes sense.